playbooks/deploy-ocp-hybrid-multinode.yml

```diff
@@ -60,16 +60,51 @@
 # - This playbook assumes the hypervisor and bastion hosts are pre-installed and ready.
 # - Test in a non-production environment before deploying.
 ---
+- name: Render nodes variables
+  hosts: nodes
+  gather_facts: false
+  tasks:
+    - name: Render nodes variables
+      delegate_to: bastion
+      ansible.builtin.set_fact:
+        network_config: "{{ network_config_string | from_yaml }}"
+        mac_interface_map: "{{ mac_interface_map_string | from_yaml }}"
```

Comment:
Why is this needed if we already have these variables?

Reply:
When we store complex yaml structure in vault hasycorp convert them to string. So we need to covert it back to yaml.